### PR TITLE
EG-1250 Implement lists in value stanzas

### PR DESCRIPTION
**Pull request by woollett-r**

### CODE REVIEW

File: test/core/models/ocelot/stanzas/ValueStanzaSpec.scala [hunk 1]
```diff
@@ -99,29 +102,47 @@ class ValueStanzaSpec extends BaseSpec {
...
       stanza.values.length shouldBe 2
-      stanza.values(0) shouldBe Value(Scalar, pageNameLabel, pageName)
-      stanza.values(1) shouldBe Value(Scalar, pageUrlLabel, pageUrl)
+      stanza.values.head shouldBe Value(ScalarType, pageNameLabel, pageName)
+      stanza.values(1) shouldBe Value(ScalarType, pageUrlLabel, pageUrl)
     }
 
-    "serialise to json" in {
-      val stanza: ValueStanza = ValueStanza(List(Value(Scalar, "LabelName", "/")), Seq("4"), true)
+    "serialize to json" in {
```
Comment (RoheelArshad) [inline test/core/models/ocelot/stanzas/ValueStanzaSpec.scala:137]: Can maybe use a better description here. Could mention "scalar" here like you've done with the "list" tests below

File: test/core/models/ocelot/stanzas/ValueStanzaSpec.scala [hunk 2]
```diff
@@ -99,29 +102,47 @@ class ValueStanzaSpec extends BaseSpec {
...
+         |  "stack": ${stack}
+         |}
+    """.stripMargin
+    )
+    .as[JsObject]
+
   "ValueStanza" must {
 
-    "deserialise from json" in {
+    "deserialize from json" in {
```
Comment (RoheelArshad) [inline test/core/models/ocelot/stanzas/ValueStanzaSpec.scala:125]: Again, mention "scalar" in description, easier to understand what this test is specific for

File: test/core/models/ocelot/stanzas/ValueStanzaSpec.scala (hunk 1 above)
Comment (RoheelArshad) [inline test/core/models/ocelot/stanzas/ValueStanzaSpec.scala:137]: lol, so we going for "s" now haha

File: test/core/models/ocelot/stanzas/ValueStanzaSpec.scala [hunk 3]
```diff
@@ -99,29 +102,47 @@ class ValueStanzaSpec extends BaseSpec {
...
+    "serialize to json" in {
+      val stanza: ValueStanza = ValueStanza(List(Value(ScalarType, "LabelName", "/")), Seq("4"), true)
       val expectedJson: String = """{"values":[{"type":"scalar","label":"LabelName","value":"/"}],"next":["4"],"stack":true}"""
       val json: String = Json.toJson(stanza).toString
       json shouldBe expectedJson
     }
 
-    "serialise to json from a Stanza reference" in {
-      val stanza: Stanza = ValueStanza(List(Value(Scalar, "LabelName", "/")), Seq("4"), true)
+    "serialize to json from a Stanza reference" in {
```
Comment (RoheelArshad) [inline test/core/models/ocelot/stanzas/ValueStanzaSpec.scala:144]: same as above

File: test/core/models/ocelot/stanzas/ValueStanzaSpec.scala (hunk 2 above)
Comment (woollett-r) [reply test/core/models/ocelot/stanzas/ValueStanzaSpec.scala:125]: Done

File: test/core/models/ocelot/stanzas/ValueStanzaSpec.scala (hunk 1 above)
Comment (woollett-r) [reply test/core/models/ocelot/stanzas/ValueStanzaSpec.scala:137]: Done

Comment (woollett-r) [reply test/core/models/ocelot/stanzas/ValueStanzaSpec.scala:137]: Yes, as defined by the Shorter Oxford English dictionary.

File: test/core/models/ocelot/stanzas/ValueStanzaSpec.scala (hunk 3 above)
Comment (woollett-r) [reply test/core/models/ocelot/stanzas/ValueStanzaSpec.scala:144]: Done

**Review by RoheelArshad — APPROVED** [review]

**Pull request merged**